### ISSUE TITLE
Add size change to GitHub Check summary

### ISSUE THREAD

**Issue by USSRLivesOn** [preactjs/compressed-size-action#22]
Currently, the GitHub Check just displays whether or not the action itself ran successfully:
<img width="683" alt="Screen Shot 2020-05-03 at 10 33 51" src="https://user-images.githubusercontent.com/234661/80921175-96b76b80-8d29-11ea-8436-f3db0ece1bcc.png">

It would be nice to have a more informative summary, perhaps including the change in size, like done in https://github.com/siddharthkp/bundlesize#build-status-for-github:
<img width="785" alt="68747470733a2f2f63646e2e7261776769742e636f6d2f7369646468617274686b702f62756e646c6573697a652f6d61737465722f6172742f7374617475732e706e67" src="https://user-images.githubusercontent.com/234661/80921252-16453a80-8d2a-11ea-9cbc-e0858261152b.png">

I know a lot of the logic already exists for the comment version of the action, and it would be great if it could be ported to the GitHub Checks version as well.

**Comment by developit** [comment]
Yes! I 100% want this, been meaning to find some time to implement it.

**Comment by swissspidy** [comment]
See also https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/
